README.md

```diff
@@ -11,8 +11,11 @@ More user relevant information about the firmware are on the wiki page at: https
 * [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
 * a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware:En:Howto) / [de](https://wiki.freifunk.net/Berlin:Firmware:Howto)) on router configuration
 
-For questions write a mail to <berlin@berlin.freifunk.net>, open a discussion with 
-@freifunk-berlin/firmware or come to our weekly meetings.
+For questions write a mail to our public mailinglist <berlin@berlin.freifunk.net>, open a discussion with
+@freifunk-berlin/firmware or come to our weekly meetings. For futher contact information have a look at https://berlin.freifunk.net/contact/
```

Comment:
is it really "futher"?

Probably also change line-length to 80 char, as discussed in https://github.com/freifunk-berlin/firmware/pull/825#discussion_r449887475

Reply:
Corrected.